tests/wpt/web-platform-tests/html/browsers/history/the-location-interface/location_origin.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  test(function () {
+    assert_equals(
+      location.origin,
+      location.protocol + '//' + location.host,
```

Comment:
The serialization code includes the the port number, but location.host doesn't. How does this test pass?